common-codegen/src/main/resources/strTemplate/stringType.stg

```diff
@@ -0,0 +1,38 @@
+template(input) ::=
+<<
+package <input.packageName>;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.lang.String;
+import java.util.Arrays;
+import java.util.Optional;
+import javax.annotation.Generated;
+
+
+@Generated(
+    value = "io.vrap.rmf.codegen.common.generator.model.codegen.inputTransformer",
+    comments = "https://github.com/vrapio/rmf-codegen"
+)
+public enum <input.simpleClassName> {
+
+  <input.enumJsonNames:{ item |<developEnum(item)>};separator=",\n\n">;
+
+  private final String jsonName;
+
+  private <input.simpleClassName>(final String jsonName) {
+    this.jsonName = jsonName;
+  }
+
+  public String getJsonName() {
+     return jsonName;
+  }
+
+  public static Optional\<<input.simpleClassName>\> findEnumViaJsonName(String jsonName) {
+    return Arrays.stream(values()).filter(t -> t.getJsonName().equals(jsonName)).findFirst();
+  }
+}
+
+>>
+developEnum(enumLateral) ::=
```

Comment:
What does `develop`mean in this context?

Reply:
develop here means extend all the enumeration values, just a suggestion, don't know if that name is convenient 😢